### ISSUE TITLE
[Notice] Calling for maintainers for the package

### ISSUE THREAD

**Issue by ahmednfwela** [4D-Technologies/openidconnect_flutter#32]
today, I forked the package to https://github.com/Bdaya-Dev/openidconnect_flutter/
and published a new variant on [pub.dev](https://pub.dev/packages/bdaya_openidconnect)

to use our variant, you need to replace 
```dart
import 'package:openidconnect/openidconnect.dart';
``` 
with 
```dart
import 'package:bdaya_openidconnect/openidconnect.dart';
```

and pubspec: `openidconnect: x.x.x` with `bdaya_openidconnect: ^1.0.0`

## Motivation

`OAuth` is used for authentication in every major website nowadays, and not always you will find a flutter package for it, and even if you do, they don't follow the same standards.
we have been looking for a package that does exactly that. unify the OAuth experience, and [we looked everywhere](https://pub.dev/packages?q=oauth) but couldn't find a package that offered everything, some didn't support all platforms, some only supported launching popups on specific platforms, and some were outright broken.

That's why we decided to take over this package and make it the perfect OAuth/OIDC package.

if the owner @jhancock4d is willing to transfer to us the original repo that would be great.

## Progress

This is how the current package looks in pub.dev

![image](https://user-images.githubusercontent.com/63286031/231279049-90470a79-8c8d-4a28-ad8c-e28c962178a7.png)

vs our variant

![image](https://user-images.githubusercontent.com/63286031/231279118-3617a70d-a03e-431d-a5a8-7123945c5ccb.png)

as you can see we fixed the platform banners, improved score and fixed some bugs on web.

we will be adding a lot more in the near future, so stay tuned !

## Maintainers

we are going to use this package for all our internal projects that require OAauth, so we are going to fix as many [bugs](https://github.com/4D-Technologies/openidconnect_flutter/issues) as we can and if anyone wants their [PR](https://github.com/4D-Technologies/openidconnect_flutter/pulls) merged, you are welcome to @ me and I will merge it against our fork

**Comment by jhancock4d** [comment]
Sorry to all.  Github was not notifying me for some reason about these issues nor the pull requests. I've started wading through.

I'm happy to take a pull request to merge the code. There are a few comments on some of the pull requests that don't seem to be applicable or are against best practices.

**Comment by jhancock4d** [comment]
You're added.

Please be careful to make sure that it continues to follow the standards (OpenIdConnect not oAuth which is way behind the OpenidConnect standard) and doesn't violate best practices. Examples are interactive flow has been depreciated as not secure so I intentionally do not support it for that reason. Google and others now fully support Code Flow.

Thanks for the help!

**Comment by ahmednfwela** [comment]
Thanks! I will be sure to request your review for every PR, since I don't have access to publish on pub.dev anyway. (would be great if we had an automated way to publish the package tho)

**Comment by jhancock4d** [comment]
I'll see if I have some time to setup a github action. But that would be the easiest way I would think.

**Comment by ahmednfwela** [comment]
Great! I will close this issue meanwhile

**Comment by radicand** [comment]
FWIW today I platform-detect and leverage either appauth for mobile or this library for web. The payloads are similar enough that it's a minor implementation detail. That said, if you were to bundle in native functionality this way, that would simplify things.

**Comment by jhancock4d** [comment]
Agreed.  Just need to find time. If you're open to pushing a PR that switches to using appauth for ios and android directly in the code, I'd welcome it.

**Comment by ahmednfwela** [comment]
I will open a new issue to continue tracking this discussion

**Comment by ahmednfwela** [comment]
Opened #33
